packages/liveblocks-react/src/factory.tsx

```diff
@@ -630,6 +652,42 @@ export function createRoomContext<
     );
   }
 
+  function usePresence<T>(
+    selector: (presence: PresenceSnapshot<TPresence, TUserMeta>) => T,
+    isEqual?: (a: unknown, b: unknown) => boolean
+  ): T {
+    type Snapshot = PresenceSnapshot<TPresence, TUserMeta>;
+
+    const room = useRoom();
+
+    const subscribe = React.useCallback(
+      (onPresenceChange: () => void) => {
+        const unsub1 = room.subscribe("my-presence", onPresenceChange);
+        const unsub2 = room.subscribe("others", onPresenceChange);
+        return () => {
+          unsub1();
+          unsub2();
+        };
+      },
+      [room]
+    );
+
+    const getSnapshot = React.useCallback(
+      (): Snapshot => room.getPresenceSnapshot(),
+      [room]
+    );
+
+    const getServerSnapshot = getSnapshot;
```

Comment:
I am guessing this is just to fit the function definition's naming in the [doc](https://reactjs.org/docs/hooks-reference.html#usesyncexternalstore)?

Reply:
Correct! 👍